tests/code/test_transcripts.py

```diff
@@ -190,23 +194,24 @@ def testingTotalHours():
 
 @pytest.mark.integration
 def teardown_module():
-
-    testingTrainingEvent = Event.get(Event.name == "Test Training Event")
-    deleteEvent(testingTrainingEvent)
-    assert Event.get_or_none(Event.id == testingTrainingEvent) is None
-    # delete bonner
-    testingBonnerEvent = Event.get(Event.name == "Test Bonner Event")
-    deleteEvent(testingBonnerEvent)
-    assert Event.get_or_none(Event.id == testingBonnerEvent) is None
-    # delete courses
-    testingCourse = Course.get(Course.courseName == "Test Course")
-    testingCourse.delete_instance(recursive = True, delete_nullable = True)
-    assert Course.get_or_none(Course.id == testingCourse.id) is None
-    # delete program
-    testingProgramEvent = Event.get(Event.name == "Test program Event")
-    deleteEvent(testingProgramEvent)
-    assert Event.get_or_none(Event.id == testingProgramEvent) is None
-    # delete user
-    user = User.get(User.username == "namet")
-    user.delete_instance(recursive = True, delete_nullable = True)
-    assert User.get_or_none(User.username == "namet") is None
+    with app.app_context():
```

Comment:
same comment as above. Only use this where it is needed.

Reply:
Can you please check again? If I remove it, my tests fail.